cas-server-core-services/src/main/java/org/jasig/cas/services/DefaultRegisteredServiceUsernameProvider.java

```diff
@@ -13,11 +13,12 @@
  */
 public final class DefaultRegisteredServiceUsernameProvider implements RegisteredServiceUsernameAttributeProvider {
     private static final long serialVersionUID = 5823989148794052951L;
-    private final transient Logger logger = LoggerFactory.getLogger(this.getClass());
+//    private final transient Logger logger = LoggerFactory.getLogger(this.getClass());
```

Comment:
Remove the commented line. There is no need for comment things out. Source control and git tracks history for you much easier.